[BUG] `aligned_resource_adaptor.hpp` doesn't compile

`rmm/mr/aligned_resource_adaptor.hpp` fails to compile with this error:

```
aligned_resource_adaptor.hpp:212:39: error: 'align_up' is not a member of 'rmm::mr::detail'; did you mean 'rmm::detail::align_up'
```

--- a/include/rmm/mr/device/aligned_resource_adaptor.hpp
+++ b/include/rmm/mr/device/aligned_resource_adaptor.hpp
@@ -209,7 +209,7 @@ class aligned_resource_adaptor final : public device_memory_resource {
    */
   std::size_t upstream_allocation_size(std::size_t bytes) const
   {
-    auto const aligned_size = detail::align_up(bytes, allocation_alignment_);
+    auto const aligned_size = rmm::detail::align_up(bytes, allocation_alignment_);
     return aligned_size + allocation_alignment_ - rmm::detail::CUDA_ALLOCATION_ALIGNMENT;
   }
 

